UPGRADING.md

```diff
@@ -190,6 +190,15 @@ This "clean up" phase can happen at the same time as your initial deployment but
       end
     end
     ```
+4. Remove `otp_secret_encryption_key` from the model setup. This also assumes you successfully ran the rake task in step 1.
```

Comment:
A nitpick: could you switch this to use the `1.` to match the other bullets above to let Markdown handle the numbering?
```suggestion
1. Remove `otp_secret_encryption_key` from the model setup. This also assumes you successfully ran the rake task in step 1.
```

Reply:
Of course!